trello_api.py

```diff
@@ -0,0 +1,92 @@
+from flask import session
+from classes import todoStatus, todoItem
+import os, requests, json, logging
+
+API_PREFIX = 'https://api.trello.com/1/'
+
+API_KEY = os.getenv('API_KEY')
+API_TOKEN = os.getenv('API_TOKEN')
+# API_PARAMS was getting appended with additional parameters, so had to set variable in each API function
+API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
+board = os.getenv('BOARD_ID')
+headers = {"Accept": "application/json"}
```

Comment:
The Accept header means what format you want the response to be in. 

Since you're only using this for a PUT request, did you mean to set "Content-Type" instead? That specifies the format of your request body.

But I think Trello accepts and returns JSON by default.